features/step_definitions/work_steps.rb

```diff
@@ -74,6 +73,7 @@
     click_button("Post Without Preview")
   end
   Work.tire.index.refresh
```

Comment:
Something for a future refactor - move these two lines that are repeated together in all these step definitions into a "refresh all work counts and indexes" method, but let's not complicate this one.